docs/code_ref/lightcurve.rst

```diff
@@ -3,6 +3,11 @@
 SunPy lightcurve
 ================
 
+.. warning::
+
+   The lightcurve submodule has been replaced with the `sunpy.timeseries`
+   module. It will be removed in a future version.
```

Comment:
for reference, the code still exists so people might need to look up the docs. The sunpy.timeseries is a link.